docker-bake.hcl

```diff
@@ -19,11 +19,19 @@ target "bench" {
     context = "images/bench"
     target = "bench"
     tags = ["frappe/bench:latest"]
+    platforms = [
```

Comment:
Add platforms directive to “default-args” target to prevent repetition

Reply:
Done. Good Idea. Thanks. It eliminates the repetitions.

As expected, it doesn't change the outcome of the build.